app/src/main/kotlin/com/camerasecuritysystem/client/CameraActivity.kt

```diff
@@ -26,6 +23,7 @@ class CameraActivity : AppCompatActivity() {
         Log.d(TAG, "onCreate: started")
     }
 
+    @SuppressLint("SetTextI18n")
```

Comment:
This could be replaced with:
```diff
diff --git a/app/src/main/kotlin/com/camerasecuritysystem/client/CameraActivity.kt b/app/src/main/kotlin/com/camerasecuritysystem/client/CameraActivity.kt
index 0aefa0c..7f63d9e 100644
--- a/app/src/main/kotlin/com/camerasecuritysystem/client/CameraActivity.kt
+++ b/app/src/main/kotlin/com/camerasecuritysystem/client/CameraActivity.kt
@@ -23,7 +23,6 @@ class CameraActivity : AppCompatActivity() {
         Log.d(TAG, "onCreate: started")
     }

-    @SuppressLint("SetTextI18n")
     fun getIncomingIntent() {
         Log.d(TAG, "getIncomingIntent: checking incoming intents")

@@ -35,7 +34,7 @@ class CameraActivity : AppCompatActivity() {

             if (mode !== CameraMode.DASHCAM) {
                 val textView: TextView = findViewById(R.id.modeText)
-                textView.text = "$mode coming soon..."
+                textView.text = String.format(resources.getString(R.string.coming_soon), mode)
             }
         }
     }
diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
index 65c8457..7e51225 100644
--- a/app/src/main/res/values/strings.xml
+++ b/app/src/main/res/values/strings.xml
@@ -44,6 +44,9 @@
     <!--Celsius-->
     <string name="temperature_suffix">\u2103</string>

+    <!-- Camera activity -->
+    <string name="coming_soon">%s coming soon…</string>
+
     <!-- Validation errors -->
     <string name="err_not_empty">%1$s cannot be empty!</string>
     <string name="err_invalid">%1$s is invalid!</string>
```

Reply:
I agree with your suggestion, it's much cleaner.

Reply:
Applied the change